Add information about rejecting a connection. (#457)
enisdenjo/graphql-ws: PROTOCOL.md

If the server receives more than one `ConnectionInit` message at any given time, the server will close the socket with the event `4429: Too many initialisation requests`.

> If the server wishes to reject the connection, for example after examining any tokens in the `payload`, it is recommended to close the socket with the event `4403: Forbidden`.

Reviewer: Less specific example.

```suggestion
If the server wishes to reject the connection, for example during authentication, it is recommended to close the socket with `4403: Forbidden`.
```

If the server receives more than one `ConnectionInit` message at any given time, the server will close the socket with the event `4429: Too many initialisation requests`.

> If the server wishes to reject the connection, for example after examining any tokens in the `payload`, it should
> close the socket with the event `4403: Forbidden`.

Reviewer: Don't add new lines for sentences.

```suggestion
If the server wishes to reject the connection, for example after examining any tokens in the `payload`, it should close the socket with the event `4403: Forbidden`.
```

> If the server wishes to reject the connection, for example after examining any tokens in the `payload`, it should
> close the socket with the event `4403: Forbidden`.

Reviewer: "should" is a strong word, we don't want to require a 4403 ever - this is just a recommendation.

```suggestion
If the server wishes to reject the connection attempt, it is recommended to close the socket with `4403: Forbidden`.
```

Author: sure.
